biome.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "./node_modules/@biomejs/biome/configuration_schema.json",
+  "vcs": { "enabled": true, "clientKind": "git", "useIgnoreFile": true },
+  "files": {
+    "include": ["**/*.json", "**/*.js", "**/*.mjs", "**/*.cjs", "**/*.ts", "**/*.tsx"],
+    "ignore": ["bin/", "*.min.js", "dist/", "package.json", ".vscode/"]
```

Comment:
Don't we want to format the package.json file? Little bit tempted to format the .vscode stuff as well :smile:

Reply:
Tried that but yarn somehow formats `package.json` as well.